### PR TITLE
fix: 5050 search bar styling 

### LINKED ISSUE #5050: Search bar results styling
image
image

- missing "collection" or its white 
- tested on top of explore 
on dark mode as well 
image

hover effect is also wrong - it should be same as links in footer and when active - bold

trending has this issue too
image
- should not change on hover

might be related with: 
- #4855

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5050 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image
image

### CODE REVIEW

File: styles/components/_search.scss [hunk 1]
```diff
@@ -232,7 +232,10 @@
     @include ktheme() {
       border-bottom-color: theme('placeholder-color');
       background: theme('background-color');
-      color: theme('text-color');
+      color: theme('text-color') !important;
+      &:hover {
+        background: theme('background-color') !important;
+      }
```
Comment (Jarsen136) [inline styles/components/_search.scss:238]:
```suggestion
      background: theme('background-color') !important;
      color: theme('text-color') !important;
```

Comment (Jarsen136) [reply styles/components/_search.scss:238]: I guess it would do the same and have less dumplicate code.

Comment (maderi21) [reply styles/components/_search.scss:238]: there is hover effect coming from styling of dropdowns in navbar which I'm overwriting here without changing the rest of the navbar. without this, the navbar hover effect gets applied here as well

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 44.37 USD/KSM ~ 0.451 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0x7ab3d04f011a98a011b8f84cd36746c7572a30bad2e7cc628fd4635b47fed2d6](https://kusama.subscan.io/extrinsic/0x7ab3d04f011a98a011b8f84cd36746c7572a30bad2e7cc628fd4635b47fed2d6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by maderi21** [comment]
> 😍 Perfect, I’ve sent the payout 💵 $20 @ 44.37 USD/KSM ~ 0.451 $KSM 🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa) 🔗 [0x7ab3d04f011a98a011b8f84cd36746c7572a30bad2e7cc628fd4635b47fed2d6](https://kusama.subscan.io/extrinsic/0x7ab3d04f011a98a011b8f84cd36746c7572a30bad2e7cc628fd4635b47fed2d6)
> 
> 🪅 Let’s grab another issue and get rewarded! 🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

thanks, however it looks like it failed 
image

**Pull request merged**